features/lege-waarden.feature

```diff
@@ -156,7 +215,27 @@ Functionaliteit: Leveren van lege waarden
       Dan heeft de response een persoon met een 'kind' met een leeg 'naam' object
       # "kinderen": [{ "naam": {} }]
 
-    Scenario: Kind met geboorteplaats onbekend geeft geboorte als leeg object zonder plaats veld
+    @gba
+    Scenario: Kind heeft onbekend geboorteplaats en onbekend geboorteland
+      Gegeven het systeem heeft een persoon met de volgende gegevens
+      | naam                | waarde    |
+      | burgerservicenummer | 555550003 |
+      En de persoon heeft een 'kind' met alleen de volgende 'geboorte' gegevens
+      | naam                   | waarde   |
+      | datum                  | 20070421 |
+      | geboorteplaats (03.20) | 0000     |
+      | geboorteland (03.30)   | 0000     |
+      Als personen wordt gezocht met de volgende parameters
+      | naam                | waarde                          |
+      | type                | RaadpleegMetBurgerservicenummer |
+      | burgerservicenummer | 555550003                       |
+      | fields              | kinderen.geboorte.plaats        |
+      Dan heeft de response een persoon met een 'kind' met de volgende 'geboorte' gegevens
+      | naam        | waarde |
+      | plaats.code | 0000   |
+      | land.code   | 0000   |
```

Comment:
Waarom zou je hier land.code uitleveren? Daar wordt in de fields toch niet om gevraagd.

Reply:
land moet niet geleverd worden want is niet gevraagd met fields